config/__init__.py

```diff
@@ -0,0 +1,20 @@
+from django.utils.version import get_version
+
+VERSION = (1, 8, 0, 'alpha', 0)
```

Comment:
I don't understand this 1.8.0alpha business; not to mention that Django 1.8 is now officially released.